src/content/deployment/android.md

```diff
@@ -296,66 +302,81 @@ check out the official [Android documentation][multidex-docs].
+```groovy title="[project]/android/app/build.gradle"
+android {
+    namespace = "com.example.[project]"
+    // Any value starting with "flutter." gets its value from
+    // the Flutter Gradle plugin.
+    // To change from these defaults, make your changes in this file.
+    [!compileSdk = flutter.compileSdkVersion!]
+    ndkVersion = flutter.ndkVersion
+
+    ...
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        [!applicationId = "com.example.[project]"!]
+        // You can update the following values to match your application needs.
+        [!minSdk = flutter.minSdkVersion!]
+        [!targetSdk = flutter.targetSdkVersion!]
+        // These two properties use values defined elsewhere in this file.
+        // You can set these values in the property declaration
+        // or use a variable.
+        [!versionCode = flutterVersionCode.toInteger()!]
+        [!versionName = flutterVersionName!]
+    }
+
+    buildTypes {
+        ...
+    }
+}
+```
 
-#### Under the `defaultConfig` block
+### Properties to adjust in build.gradle
 
-`applicationId`
-: Specify the final, unique [application ID][].
-  
-`minSdk`
-: Specify the [minimum API level][] on which you designed the app to run.
-  Defaults to `flutter.minSdkVersion`.
+| Property             | Purpose                                                                                                                                                                                                            | Default Value               |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| `compileSdk`         | The Android API level with which your app should be compiled.                                                                                                                                                      | `flutter.compileSdkVersion` |
+| `defaultConfig`      |  |  |
+| `.applicationId`     | The final, unique [application ID][] that identifies your app.                                                                                                                                                     |                             |
+| `.minSdk`            | The [minimum Android API level][] for which you designed the app to run.                                                                                                                                           | `flutter.minSdkVersion`     |
+| `.targetSdk`         | The target Android API level for which you designed the app to run.                                                                                                                                                | `flutter.targetSdkVersion`  |
```

Comment:
https://stackoverflow.com/questions/26694108/what-is-the-difference-between-compilesdkversion-and-targetsdkversion
This stackoverflow question does a good job explaining the difference